subprojects/docs/src/docs/userguide/publishingIvy.adoc

```diff
@@ -133,25 +133,25 @@ The generated Ivy module descriptor file contains an `&lt;info&gt;` element that
 * `status` - api:org.gradle.api.Project#getStatus()[]
 * `branch` - (not set)
 
-Overriding the default identity values is easy: simply specify the `organisation`, `module` or `revision` attributes when configuring the `IvyPublication`. The `status` and `branch` attributes can be set via the `descriptor` property (see api:org.gradle.api.publish.ivy.IvyModuleDescriptorSpec[]). The `descriptor` property can also be used to add additional custom elements as children of the `&lt;info&gt;` element.
+Overriding the default identity values is easy: simply specify the `organisation`, `module` or `revision` attributes when configuring the api:org.gradle.api.publish.ivy.IvyPublication[]. The `status` and `branch` attributes can be set via the `descriptor` property (see api:org.gradle.api.publish.ivy.IvyModuleDescriptorSpec[]). The `descriptor` property can also be used to add additional custom elements as children of the `&lt;info&gt;` element.
```

Comment:
Is `organisation` correct here? I'm just wondering why it's not US spelling.

Reply:
It uses the same spelling as the Ivy descriptor: